homeassistant/components/light/mqtt.py

```diff
@@ -51,13 +51,17 @@
 CONF_WHITE_VALUE_SCALE = 'white_value_scale'
 CONF_WHITE_VALUE_STATE_TOPIC = 'white_value_state_topic'
 CONF_WHITE_VALUE_TEMPLATE = 'white_value_template'
+CONF_ON_COMMAND_TYPE = 'on_command_type'
 
 DEFAULT_BRIGHTNESS_SCALE = 255
 DEFAULT_NAME = 'MQTT Light'
 DEFAULT_OPTIMISTIC = False
 DEFAULT_PAYLOAD_OFF = 'OFF'
 DEFAULT_PAYLOAD_ON = 'ON'
 DEFAULT_WHITE_VALUE_SCALE = 255
+DEFAULT_ON_COMMAND_TYPE = "LAST"
+
+VALUES_ON_COMMAND_TYPE = ["FIRST", "LAST", "BRIGHTNESS"]
```

Comment:
What the advantage to use capital letters here?

Reply:
None.  ON and OFF were capitalized so that's what I followed.  It doesn't make a big difference if they're upper, lower, or case insensitive.

Reply:
Those are configuration setting and all those are lower case. `ON` and `OFF` are payloads.